.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - make init
 script:
   - make build test lint coverage
+  - make docker-all
```

Comment:
Ok. Both are fair points. Though I look forward to a point where we can drop one path or the other.